src/main.py

```diff
@@ -10,17 +12,33 @@
 app.include_router(router)
 
 
-def create_bot():
+def create_bot() -> Application:
     """Создать бота."""
     bot_app = ApplicationBuilder().token(settings.BOT_TOKEN).build()
     bot_app.add_handler(CommandHandler("start", start))
     return bot_app
 
 
-async def start_bot():
+async def start_bot() -> None:
     """Запустить бота."""
     bot_app = create_bot()
     await bot_app.updater.initialize()
     await bot_app.initialize()
-    await bot_app.updater.start_polling()
+    if settings.BOT_WEBHOOK_MODE:
+        webhook_settings = get_bot_webhook_settings()
+        await bot_app.updater.start_webhook(**webhook_settings)
```

Comment:
```suggestion
        await bot_app.bot.start_webhook(**webhook_settings)
```
Кажется так можно сделать, если да, то лучше так и поступить

Reply:
Неактуально. start_webhook() запускает веб-сервер.